src/utils/payments/pawapay/types.ts

```
@@ -0,0 +1,69 @@
+  msisdn?: string;
+  language?: 'EN' | 'FR';
+  country?: string;
+  reason?: string;
+  metadata?: Array<{
+    fieldName: string;
+    fieldValue: string;
+    isPII?: boolean;
+  }>;
+}
+
+interface PaymentPageSessionResponse {
```

ESLint

[error] Use a `type` instead of an `interface`.

(ts/consistent-type-definitions)